pyproject.toml

```diff
@@ -63,7 +63,7 @@ emit_code_comments = "True"
 
 [tool.cibuildwheel]
 build-frontend = "build"
-enable = ["cpython-freethreading"]
+enable = ["cpython-freethreading", "cpython-prerelease"]
```

Comment:
I don't think we want to enable this. This will build wheels for beta releases which are not ABI stable and may break when installed on a different release.

Reply:
```suggestion
enable = ["cpython-freethreading"]
```